llvm/test/tools/llvm-reduce/run-all-delta-in-child.ll

```diff
@@ -0,0 +1,53 @@
+; UNSUPPORTED: target={{.-windows.}}
+; RUN: llvm-reduce --run-delta-in-child --test FileCheck --test-arg --check-prefixes=CHECK-INTERESTINGNESS --test-arg %s --test-arg --input-file %s -o %t
+; RUN: FileCheck --check-prefixes=CHECK-FINAL --input-file=%t %s
+
```

Comment:
So does this test demonstrate a crash in a reduction? I'm confused about what this is showing